src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray`1.cs

```diff
@@ -875,23 +886,25 @@ public ImmutableArray<T> RemoveAll(Predicate<T> match)
             Requires.NotNull(match, "match");
 
             if (self.IsEmpty)
-            {
-                return new ImmutableArray<T>(self.array);
-            }
+                return self;
```

Comment:
Even if we change this to ```return self;``` instead of ```return new ImmutableArray<T>(self.array);```, we should still keep the braces that keep the formatting consistent as much as possible with the rest of the project.